### PR TITLE
Update metrics.md #229

### PR DESCRIPTION
**Pull request by hnjolles1**

### CODE REVIEW

File: docs/video-miners/guides/metrics-monitoring.md [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
+- Visualized in [Grafana](https://grafana.com/) 
 
-## Monitoring with the livepeer monitoring container
+## Monitoring with Docker
 
-A
-[Docker container](https://github.com/livepeer/docker-livepeer/tree/master/monitoring)
-that bundles Prometheus, Grafana and a few starter Grafana dashboard templates
-can also be used.
+You can use the [Docker container](https://github.com/livepeer/docker-livepeer/tree/master/monitoring). It bundles Prometheus, Grafana, and a few starter Grafana dashboard templates.
```
Comment (hthillman) [inline docs/video-miners/guides/metrics-monitoring.md:45]: How about 

You can use the [Docker container](https://github.com/livepeer/livepeer-monitoring) maintained by [Livepeer Video Services](https://livepeer.com) to easily start monitoring your orchestrator or transcoder.

Comment (hthillman) [reply docs/video-miners/guides/metrics-monitoring.md:45]: (notice the updated link)

**Review by hthillman — CHANGES_REQUESTED** [review]

File: docs/video-miners/guides/metrics-monitoring.md [hunk 2]
```diff
@@ -1,44 +1,45 @@
 ---
-title: Monitor Metrics
+title: Monitoring Metrics
 ---
 
-# Monitor Metrics
+This guide provides instructions on configuring metrics monitoring for orchestrators that have been [activated](/video-miners/getting-started/activation) on the Livepeer network.
```
[the remark this answers is not in the document]
Comment (hnjolles1) [reply docs/video-miners/guides/metrics-monitoring.md:5]: Hi Hunter, I asked Meredith and she confirmed it is Livepeer network. I checked the docs and it's consistent with what is in other docs. I will keep an eye out.

File: docs/video-miners/guides/metrics-monitoring.md (hunk 1 above)
Comment (hnjolles1) [reply docs/video-miners/guides/metrics-monitoring.md:45]: Changed the Link on Docker Container, and added the link to Livepeer Video Services.

File: docs/video-miners/guides/metrics-monitoring.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hthillman) [reply docs/video-miners/guides/metrics-monitoring.md:5]: Great, thank you!

File: docs/video-miners/guides/metrics-monitoring.md (hunk 1 above)
Comment (hthillman) [reply docs/video-miners/guides/metrics-monitoring.md]: awesome, thanks!

**Review by hthillman — APPROVED** [review]

**Pull request merged**